conferences/migrations/2_create_ticketing_tables.up.sql

```diff
@@ -0,0 +1,65 @@
+CREATE TABLE slot_claim (
+    id BIGSERIAL PRIMARY KEY,

```

Comment:
I used bigints for fields mirroring uint64 in go structs and ints for int then things changed and the sql stayed, i think it would be reasonable to use just int for everything, ill change it